beacon-chain/rpc/service.go

```diff
@@ -63,6 +67,7 @@ type Service struct {
 	incomingAttestation   chan *pbp2p.Attestation
 	enablePOWChain        bool
 	slotAlignmentDuration time.Duration
+	credentialError       error
```

Comment:
shouldn't this be "general" error rather than a specific error?

Reply:
I remember @prestonvanloon  suggested it to be a specific error